documentation/modules/tracing/proc-enabling-tracing-in-connect-mirror-maker-bridge-resources.adoc

```diff
@@ -64,8 +64,31 @@ spec:
   tracing:
     type: jaeger
 #...
+----
 
+.Jaeger tracer configuration for MirrorMaker 2.0
+[source,yaml,subs=attributes+]
 ----
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: KafkaMirrorMaker2
+metadata:
+  name: my-mm2-cluster
+spec:
+  #...
+  template:
+    mirrorMakerContainer:
```

Comment:
Actually, for MM2 this field is a bit different - that sould be the only difference ...

```suggestion
    connectContainer:
```